cmake/test.cmake

```diff
@@ -75,64 +75,64 @@ function(mayaUsd_add_test test_name)
             "used with PYTHON_MODULE or PYTHON_COMMAND")
     endif()
 
-    # set the working_dir
+    # set the WORKING_DIR
     if(PREFIX_WORKING_DIRECTORY)
-        set(working_dir ${PREFIX_WORKING_DIRECTORY})
+        set(WORKING_DIR ${PREFIX_WORKING_DIRECTORY})
     else()
-        set(working_dir ${CMAKE_CURRENT_SOURCE_DIR})
+        set(WORKING_DIR ${CMAKE_CURRENT_SOURCE_DIR})
     endif()
 
     # --------------
     # 2) Create test
     # --------------
 
-    set(pytest_code "")
+    set(PYTEST_CODE "")
     if(PREFIX_PYTHON_MODULE)
-        set(module_name "${PREFIX_PYTHON_MODULE}")
-        set(pytest_code "
+        set(MODULE_NAME "${PREFIX_PYTHON_MODULE}")
+        set(PYTEST_CODE "
 import sys
 from unittest import main
-import ${module_name}
-main(module=${module_name})
+import ${MODULE_NAME}
+main(module=${MODULE_NAME})
 ")
     elseif(PREFIX_PYTHON_COMMAND)
-        set(pytest_code "${PREFIX_PYTHON_COMMAND}")
+        set(PYTEST_CODE "${PREFIX_PYTHON_COMMAND}")
     else()
-        set(command ${PREFIX_COMMAND})
+        set(COMMAND ${PREFIX_COMMAND})
     endif()
 
-    if(pytest_code)
+    if(PYTEST_CODE)
         if(NOT PREFIX_NO_STANDALONE_INIT)
             # first, indent pycode
-            mayaUsd_indent(indented_pytest_code "${pytest_code}")
+            mayaUsd_indent(indented_PYTEST_CODE "${PYTEST_CODE}")
```

Comment:
Why is "indented" still lowercase?

Reply:
indented_PYTEST_CODE is a function argument. My goal for this PR was to tackle the set variables.

Reply:
Okay, but you changed the "pytest_code" to uppercase so now it looks weird being part lower and part upper.